### ISSUE TITLE
lodestar start error

### ISSUE THREAD

**Issue by yrong** [ChainSafe/lodestar#5100]
**Describe the bug**

```
(node:16540) [FST_MODULE_DEP_FASTIFY-CORS] FastifyWarning.fastify-cors: fastify-cors has been deprecated. Use @fastify/cors@7.0.0 instead.
(Use `node --trace-warnings ...` to show where the warning was created)
Eph -1/6 4.345[]                 [32minfo[39m: Lodestar network=dev, version=v1.3.0/foundry-test/5e40df4, commit=5e40df4b2924bdc6f70fe7bbf51036d4d7f3bedb
Eph -1/6 4.346[]                 [32minfo[39m: ACTIVE_PRESET == minimal preset
Eph -1/6 4.350[]                 [32minfo[39m: Connected to LevelDB database path=/tmp/snowbridge/beacon-1675599436/chain-db
Eph -1/6 4.356[]                 [32minfo[39m: Initializing beacon from a valid checkpoint state slot=0, epoch=0, stateRoot=0x6a5e872e03d7ffbb28079987f4ade5fc20199023e69acf860dbbd7cebf9f72b2, isWithinWeakSubjectivityPeriod=true
 ✖ Error: network.discv5.enr must be an instance of ENR
    at createNodeJsLibp2p (file:///Users/yangrong/Projects/snowbridge/core/node_modules/.pnpm/@lodestar+beacon-node@1.4.1_@chainsafe+blst@0.2.8/node_modules/@lodestar/beacon-node/src/network/nodejs/util.ts:40:13)
    at beaconHandler (file:///Users/yangrong/Projects/snowbridge/core/node_modules/.pnpm/@chainsafe+lodestar@1.3.0_fastify@3.15.1/node_modules/@chainsafe/lodestar/src/cmds/beacon/handler.ts:77:15)
    at Object.devHandler [as handler] (file:///Users/yangrong/Projects/snowbridge/core/node_modules/.pnpm/@chainsafe+lodestar@1.3.0_fastify@3.15.1/node_modules/@chainsafe/lodestar/src/cmds/dev/handler.ts:68:3)
```

**Expected behavior**

`beacon node starting without error`

**Steps to Reproduce**

```
"devDependencies": {
    "@chainsafe/lodestar": "1.3.0",
    ...
}
```

```
npx lodestar dev \
            --genesisValidators 8 \
            --genesisTime $timestamp \
            --startValidators "0..7" \
            --enr.ip "127.0.0.1" \
            --network.connectToDiscv5Bootnodes=false \
            --dataDir "/tmp/snowbridge/beacon-1675599436" \
            --reset \
            --terminal-total-difficulty-override 0 \
            --genesisEth1Hash $genesisHash \
            --params.ALTAIR_FORK_EPOCH 0 \
            --params.BELLATRIX_FORK_EPOCH 0 \
            --eth1=true \
            --rest.namespace="*" \
            --jwt-secret config/jwtsecret \
```

**Desktop (please complete the following information):**

- OS:  MacOS Ventura 13.2(m1)
- Version: 1.3.0

**Comment by wemeetagain** [comment]
Should be resolved with 1.4.x

**Comment by yrong** [comment]
@wemeetagain  thanks a lot for the reply!  and yes, verified works with `1.4.1` but  not with `1.4.0`, seems fixed in this pr
https://github.com/ChainSafe/lodestar/pull/5073
